pkg/aggregator/aggregator.go

```diff
@@ -319,6 +320,7 @@ func (agg *BufferedAggregator) flushSeries() {
 	series = append(series, &metrics.Serie{
 		Name:           fmt.Sprintf("datadog.%s.running", agg.agentName),
 		Points:         []metrics.Point{{Value: 1, Ts: float64(start.Unix())}},
+		Tags:           []string{fmt.Sprintf("agent_version:%s", version.AgentVersion)},
```

Comment:
I think we could either:
1. name the tag `version:`, or
2. reflect the `agent`/`cluster_agent` difference in the tag key as well: `agent_version:`/`cluster_agent_version:`

I think `1.` would be clear enough. Any thoughts? (cc @xvello as well)

Reply:
I agree, as the gauges have different names, let's go with the short `version` tag 👌

Reply:
Yes `version:` should be enough since the metric is already of the name of the agent

Reply:
👍